docs/docs/en/contributors/code-reviews.md

```diff
@@ -4,29 +4,29 @@ titleTemplate: ':title - Contribute to Tuist'
 description: Learn how to contribute to Tuist by reviewing code
 ---
 
-<h1 id="code-reviews">Code reviews</h1>
+# Code reviews {#code-reviews}
```

Comment:
This is necessary to fix the search. The way Vitepress constructs a local index is by inspecting the Markdown titles and ids. Since we moved that to heading HTML tags, we broke the local search.